bean.js

```diff
@@ -0,0 +1,11 @@
+const countBs = string => countChar(string, 'B');
+
+const countChar = (string, char) => {
+    let count = 0;
+    for (let i = 0; i < string.length; i++) {
```

Comment:
looks good, but try replacing the `for` loop with `forEach`

Reply:
You can;t loop over string so try to use it like that

`[...string].forEach()`

or 

`string.split('').forEach()`